src/pages/starter-kit/checkout/tax-install.md

```diff
@@ -38,6 +38,10 @@ magento setup:di:compile
 
 ## Configuration
 
+<InlineAlert variant="info" slots="text"/>
+
+This step requires [Adobe Commerce HTTP Client](./connect.md#connect-to-adobe-commerce) to authenticate the Commerce instance.
```

Comment:
```suggestion
This step requires the [Adobe Commerce HTTP Client](./connect.md#connect-to-adobe-commerce) to authenticate the Commerce instance.
```

Reply:
For repeated notes like this, you can use the `_includes` folder along with an `import`.

Reply:
I extracted them as `_includes`, thanks